mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -99,4 +100,10 @@ boolean hasChildren(Mailbox mailbox, char delimiter)
      * @throws MailboxException 
      */
     List<Mailbox> list() throws MailboxException;
+
+    List<MailboxAnnotation> getAnnotations(Mailbox mailbox) throws MailboxException;
```

Comment:
Ok, I'll try to explain the policy : we don't like documentation. It's often misleading, out of date, describe something that should be obvious by reading the code, etc. But sometime it's just required : usually, interfaces that should have several implementation. So most of mailbox-api project for example. MailboxMapper is not meant to be really a public thing, but it's supposed to be implemented for each backend, so explaining what methods should do is a good idea (even if tests are here to explain expected behavior better ...)